indico/modules/events/registration/fields/simple.py

```diff
@@ -110,6 +121,9 @@ def modify_post_data(cls, post_data):
 class BooleanField(RegistrationFormBillableField):
     name = 'yes/no'
     wtf_field_class = IndicoRadioField
+    mapping = {None: '',
```

Comment:
I think it's better to have it in the class level, so I can reuse it to render the dropdown in the filter dialog